kolibri/plugins/user_auth/assets/src/views/UserAuthLayout.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+
+  <div
+    ref="mainWrapper"
+    class="main-wrapper"
+    :style="mainWrapperStyles"
+  >
+
+    <div v-if="blockDoubleClicks" class="click-mask"></div>
+
+    <div
+      v-if="!loading"
+      class="scrolling-pane"
+    >
+      <CoreBanner v-if="coreBannerComponent && showDemoBanner">
+        <template #default="props">
+          <component
+            :is="coreBannerComponent"
+            :bannerClosed="props.bannerClosed"
+          />
+        </template>
+      </CoreBanner>
+
+      <KPageContainer v-if="!isAuthorized">
+        <AuthMessage />
+      </KPageContainer>
+      <KPageContainer v-else-if="error">
+        <AppError />
+      </KPageContainer>
+
+      <div
+        v-else
+        id="main"
+        role="main"
+        tabindex="-1"
+        class="main"
+      >
+        <slot></slot>
+      </div>
+    </div>
+
+    <div aria-live="polite">
+      <GlobalSnackbar />
+    </div>
```

Comment:
I'm not familiar with "polite" here and it's new to our code base; could you add a little comment along with this explaining what it does in this context? 

Also, if this is how we should be handling our snackbars, we should add this aria-live to all of them. I wonder if it would be easiest to just add the aria attribute directly in `CoreSnackbar`'s wrapping div?

Reply:
The aria-live was added by @marcellamaki  about 5 months ago to the `GlobalSnackbar`. Based on what the aria-live does, i think it should be added to the `CoreSnackbar`. Adding it there automatically adds it to the `GlobalSnackbar`. @marcellamaki any thoughts on this?

Reply:
@akolson that might work, I am trying to think through the implications, but I would suggest no matter what that it is worth recruiting @radinamatic to do some testing so we can insure this still works. 😄 

It took some trial and error to come up with placing the `aria-live="polite"` tag in the wrapping div, and the reason it was necessary is because the div itself must _always_ be present in the DOM in order for the `aria-live` interruption to work. Essentially what is happening (cc @nucleogenesis) is that having an empty div with the `aria-live` announces to the screen reader user ("politely" - meaning waiting until they are idle) that the content of that div has changed. In this case, when the snackbar is rendered, that the text associated with that snackbar. 

However, the `aria-live` cannot live _just_ on the component that must be announced itself, because then it is not always "listening" - it is only rendered along with the "changing" content. This implementation does not work across all screen readers (there are some browser-based reader, such as VoiceOver on Mac where this does work). I think it would be possible to add this `aria-live` to the `CoreSnackbar` but only if we are certain that in the course of this refactor, the `CoreSnackbar` component will always be present in DOM, even if it is not always visible. Otherwise, we should follow the same pattern and add an empty div that wraps around the snackbar (and thus will not interfere with the visual UI)

Reply:
Thanks @marcellamaki. This is very insightful. At this point I am not too sure that adding it directly to the `CoreSnackbar` would work without making tweaks to the way its rendered to the DOM